hadoop-ozone/dist/src/main/smoketest/omha/testOMHA.robot

```diff
@@ -182,4 +194,21 @@ Restart OM and Verify Ratis Logs
     # Verify that the logs match with the Leader OMs logs
     List Should Contain Sub List    ${logsAfter}        ${logsLeader}
 
+Transfer Leadership for OM
+    # Check OM write operation before failover
+    Create volume and bucket
+    Write Test File
 
+    # Find Leader OM and one Follower OM
```

Comment:
It would be good to add a test for random (old leader != new leader should be enough for random)

Reply:
Done